src/library_html5.js

```diff
@@ -184,16 +184,16 @@ var LibraryJSEvents = {
       }
       var handlerFunc = function(event) {
         var e = event || window.event;
-        writeStringToMemory(e.key ? e.key : "", JSEvents.keyEvent + {{{ C_STRUCTS.EmscriptenKeyboardEvent.key }}} );
-        writeStringToMemory(e.code ? e.code : "", JSEvents.keyEvent + {{{ C_STRUCTS.EmscriptenKeyboardEvent.code }}} );
+        stringToUTF8(e.key ? e.key : "", JSEvents.keyEvent + {{{ C_STRUCTS.EmscriptenKeyboardEvent.key }}}, 32);
```

Comment:
is this 32 (and later down 128 etc.) the size of the field in the struct? could we define a C constant in the header, add it to `struct_info.json` and then read it here, so we don't have these constants which might get out of date?